debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -1517,11 +1517,30 @@ protected void addToTransaction(String transactionId, LogMinerEventRow row, Supp
             return;
         }
 
+        final LogMinerEvent event;
+        try {
+            event = eventSupplier.get();
+        }
+        catch (DmlParserException e) {
+            switch (connectorConfig.getEventProcessingFailureHandlingMode()) {
+                case FAIL:
+                    LOGGER.error("Failed to parse SQL for event '{}'", row);
```

Comment:
Why `FAIL` and `default` branches have different message? Should not be SQL statement logged in all cases?